src/main/java/ladder/domain/Height.java

```diff
@@ -0,0 +1,33 @@
+package ladder.domain;
+
+public class Height {
+
+    private static final int MIN_HEIGHT_RANGE = 1;
+    private static final int MAX_HEIGHT_RANGE = 100;
+
+    private final int height;
+
+    public Height(String input) {
+        int number = changeInputToInteger(input);
+        validateNumberIsIntRange(number);
+        this.height = number;
+    }
+
+    private int changeInputToInteger(String input) {
+        try {
+            return Integer.parseInt(input);
+        } catch (NumberFormatException e) {
+            throw new NumberFormatException("숫자로 입력을 변환할 수 없습니다.");
+        }
+    }
+
+    private void validateNumberIsIntRange(int number) {
+        if (number < MIN_HEIGHT_RANGE || number > MAX_HEIGHT_RANGE) {
```

Comment:
미세 가독성 팁인데 꺽쇠 방향을 일치시키면 조금 더 쉽게 읽힙니다!
```suggestion
        if (number < MIN_HEIGHT_RANGE || MAX_HEIGHT_RANGE < number) {
```

Reply:
경험에서 우러나오는 이런 미세 꿀팁 개인적으로 너무 좋아합니다 ㅠㅠ 😍